scripts/setupvars/setupvars.bat

```diff
@@ -37,6 +37,13 @@ set PYTHONPATH=%INTEL_OPENVINO_DIR%\deployment_tools\model_optimizer;%PYTHONPATH
 set "PATH=%INTEL_OPENVINO_DIR%\deployment_tools\model_optimizer;%PATH%"
 )
 
+
+:: Model Downloader
+if exist %INTEL_OPENVINO_DIR%\deployment_tools\open_model_zoo\tools\downloader (
+set PYTHONPATH=%INTEL_OPENVINO_DIR%\deployment_tools\open_model_zoo\tools\downloader;%PYTHONPATH%
+set "PATH=%INTEL_OPENVINO_DIR%\deployment_tools\open_model_zoo\tools\downloader;%PATH%"
```

Comment:
"" usage is not consistent (almost everywhere with PYTHONPATH).

@asomsiko Do you know which variant is proposed by secure programming guidelines?